Fix horizontal scrolling / width on mobile

The video shows that the start page haskell.org behaves glitchy on mobile. You can scroll horizontally, but you shouldn't be able to. This doesn't result in a terrible user experience, but still is an issue that should be avoided.


https://user-images.githubusercontent.com/13085980/216581498-71a48d5f-7a4d-478f-8a8f-8b674b9851aa.mp4

I tested with Android 13 and latest Chrome and Kiwi Browser on a OnePlus 8.

I wanted to start taking a look, but `buildAndWatch` ended up in:

```
/nix/store/2md452mq5labdcxlsc5gfik97dhxabbr-haskell-org-0.1.0.0
Removing _site...
Removing _cache...
Removing _cache/tmp...
Initialising...
  Creating store...
  Creating provider...
  Running rules...
Checking for out-of-date items
Compiling
  updated 2015-11-10-Haskell-Org-Board-Resolution.pdf
  [ERROR] _cache/a98f5fd1dec8ada591e76ae4112817f9 for Hakyll.Core.Resource.Provider.MetadataCache/community.markdown/body: Store.set: invalid argument (invalid byte sequence)
```

Is there a quick fix, or is this a reason to open an issue?

--- a/site/css/hl.css
+++ b/site/css/hl.css
@@ -4629,7 +4629,7 @@ pre .hs-keyglyph {
   .glider,.glider-contain{margin:0 auto;position:relative;}
   .glider,.glider-track{transform:translateZ(0)}
   .glider-dot,.glider-next,.glider-prev{border:0;padding:0;user-select:none;outline:0}
-  .glider-contain{width:100%}
+  .glider-contain{width:100%;overflow-x: hidden;}
   .glider{overflow:auto;scrollbar-width: none; overflow-y:hidden;-webkit-overflow-scrolling:touch;-ms-overflow-style:none; justify-content: space-between; display: flex;
     align-items: flex-start;}
   .glider-track{width:100%;margin:0;padding:0;display:flex;z-index:1}

